android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.reactlibrary">
+          package="com.robinpowered.RNAudioJack">
```

Comment:
RE naming feedback, this, and other references in Android, should drop the `RN` prefix